src/appkit.cs

```diff
@@ -1040,6 +1053,30 @@ interface NSApplicationDelegate {
 		[Deprecated (PlatformName.MacOSX, 11, 0, message: "Now optional on NSApplicationDelegate.")]
 		[Export ("application:delegateHandlesKey:"), DelegateName ("NSApplicationHandlesKey"), NoDefaultValue]
 		bool HandlesKey (NSApplication sender, string key);
+
+		[Mac (12,0)]
+		[IgnoredInDelegate]
+		[Export ("applicationSupportsSecureRestorableState:")]
+		bool ApplicationSupportsSecureRestorableState (NSApplication application);
```

Comment:
The 'application' suffix is often emitted from the managed name in this case, because it's somewhat redundant (look at the other methods in this class, they mostly do the same thing) - in the UIApplicationDelegate class the logic is about UIApplication, which means that spelling out 'Application' all the time feels a bit unnecessary:

```suggestion
		bool SupportsSecureRestorableState (NSApplication application);
```